apps/meteor/ee/app/livechat-enterprise/server/methods/resumeOnHold.ts

```diff
@@ -60,7 +65,6 @@ Meteor.methods({
 		const comment = await resolveOnHoldCommentInfo(options, room, onHoldChatResumedBy);
 		(Messages as any).createOnHoldResumedHistoryWithRoomIdMessageAndUser(roomId, comment, onHoldChatResumedBy);
 
-		const updatedRoom = LivechatRooms.findOneById(roomId);
-		updatedRoom && Meteor.defer(() => callbacks.run('livechat:afterOnHoldChatResumed', updatedRoom));
+		Meteor.defer(() => callbacks.run('livechat:afterOnHoldChatResumed', room));
```

Comment:
Won't have any problems by using an outdated room instead of the updated one?

Reply:
I've modified the callbacks to work without the updated room data. So we can save this DB call here :tada: